nrf_security/Kconfig

```diff
@@ -861,7 +861,7 @@ config MBEDTLS_ECJPAKE_C
 	select CC310_MBEDTLS_ECJPAKE_C if CC310_MBEDTLS_ECP_C
 	select OBERON_MBEDTLS_ECJPAKE_C if OBERON_MBEDTLS_ECP_C
 	select VANILLA_MBEDTLS_ECJPAKE_C if VANILLA_MBEDTLS_ECP_C
-	default y
+	default n
```

Comment:
```suggestion
```

No need to specify `default n` as that is the default when nothing is specified.

So line can be removed.

Reply:
Fixed